doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
 event:
 
     somePromise.then(function(res) {
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+'unhandledRejection':
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
+    }
+
+    var resource = new SomeResource();
+    // no .catch or .then on resource.loaded for at least a turn
```

Comment:
I think the solution is to create a document on io's async model that explains what a microtask is, a macrotask is and a queue of tasks is. Otherwise I really can't see how we can expect people to understand this terminology.